webrtc-stats.html

```diff
@@ -2695,8 +2695,10 @@ <h2>
               <dd>
                 <p>
                   The total number of samples that are concealed samples. A concealed sample is a
-                  sample that is based on data that was synthesized to conceal packet loss and does
-                  not represent incoming data.
+                  sample that is based on data that was synthesized to conceal missing samples
+                  during playout. Concealed samples may be inserted when a packet is lost
+                  (reported in <a>packetsLost</a>) or the packet arrives too late to be played out
+                  (reported in <a>packetsDiscarded</a>).
```

Comment:
Is it too late to change the name? I find the name "concealed" very confusing - since the samples we're counting is samples that were synthesized, not the (unknown) samples they were synthesized to conceal.

If we want to keep the "concealed" name, rewrite as: "A concealed sample is a sample that was replaced with synthesized samples generated locally before being played out. Examples of samples that have to be concealed are samples from lost packets (reported in packetsLost) or samples from packets that arrive too late to be played out (reported in packetsDiscarded).

The "examples of" language allows us to use the counter for samples that are removed for other reasons, such as being identifies as Doritos munching, keyboard clicks or sounds too loud to be played out (if that makes sense).

Reply:
It is too late to change the name (this is already shipped in Chrome). Update the definition as you suggested, making it more flexible, though if we had different reasons to conceal we might need more stats to help distinguish between "concealing" and other reasons in the future.